# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mp4parse_capi/examples/dump.rs
# Commit: 1d5a44ae7207
# Full Hash: 1d5a44ae72071f079dcb00b62419c2ddd3c9f12c
# Author: Norisz Fay <nfay@mozilla.com>
# Date: 2023-01-27 21:11 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing build bustages on DecoderData.cpp CLOSED TREE
# ==============================================================================

diff -r 83609f9575ac -r 1d5a44ae7207 third_party/rust/mp4parse_capi/examples/dump.rs
--- a/third_party/rust/mp4parse_capi/examples/dump.rs	Fri Jan 27 19:06:16 2023 +0000
+++ b/third_party/rust/mp4parse_capi/examples/dump.rs	Fri Jan 27 21:11:42 2023 +0200
@@ -1,4 +1,11 @@
-use log::info;
+extern crate mp4parse;
+extern crate mp4parse_capi;
+
+#[macro_use]
+extern crate log;
+
+extern crate env_logger;
+
 use mp4parse::ParseStrictness;
 use mp4parse_capi::*;
 use std::env;
@@ -112,9 +119,7 @@
                         }
                     }
                 }
-                Mp4parseTrackType::Video
-                | Mp4parseTrackType::Picture
-                | Mp4parseTrackType::AuxiliaryVideo => {
+                Mp4parseTrackType::Video => {
                     let mut video_info = Mp4parseTrackVideoInfo::default();
                     match mp4parse_get_track_video_info(parser, i, &mut video_info) {
                         Mp4parseStatus::Ok => {